apps/hosts/admin.py

```diff
@@ -1,3 +1,20 @@
 from django.contrib import admin
 
-# Register your models here.
+from base.admin import TimeStampedAdmin
+
+from .models import (ChallengeHostAdmin,
```

Comment:
wrong import. model name concerned here are `ChallengeHostTeam` and `ChallengeHost`